solutions/src/main/java/ru/sberbank/school/task02/FxConversionServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+import java.math.BigDecimal;
+
+
+public class FxConversionServiceImpl implements FxConversionService{
+
+    ExternalQuotesService externalQuotesService;
+
+    FxConversionServiceImpl(ExternalQuotesService externalQuotesService){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1208769466>*